### PR TITLE
 映射文档 No. 370

### PR DESCRIPTION
**Pull request by co63oc**
https://github.com/PaddlePaddle/PaConvert/issues/112

370 torch.nn.functional.dropout1d

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.dropout1d.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+torch.nn.functional.dropout1d(input, p=0.5, training=True, inplace=False)
+```
+
+### [paddle.nn.functional.dropout](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/dropout_cn.html)
+
+```python
+paddle.nn.functional.dropout(x, p=0.5, axis=None, training=True, mode='upscale_in_train', name=None)
+```
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.dropout1d.md:16]: 可以再详细一点解释下两者细微差异，torch对于dropout1d/dropout2d/dropout3d，是将某个Channel以一定概率全部置0，paddle是所有元素以一定概率置0
然后再说下 但该差异一般不影响网络训练效果

Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.dropout1d.md:16]: 已修改

**Review by zhwesky2010 — COMMENTED** [review]
dropout2d、dropout3d的文档也顺便一起补上吧

**Review by zhwesky2010 — APPROVED** [review]

**Comment by co63oc** [comment]
> dropout2d、dropout3d的文档也顺便一起补上吧

增加 torch.nn.functional.dropout2d/dropout3d
https://github.com/PaddlePaddle/docs/pull/6161
